.travis.yml

```diff
@@ -7,3 +7,4 @@ rvm:
   - 2.1.0
   - 2.2.0
   - 2.3.0
+before_script: bundle exec rake build && [[ -z $(git status -s) ]]
```

Comment:
What about:

```
before_script: bundle exec rake build && git status -s && [[ -z $(git status -s) ]]
```

Would that work? Does the `before_script` require a 0 exit code and does `git status` give a 0 exit code whether or not there are changes detected?